lib/core/Header.js

```diff
@@ -17,7 +18,7 @@ class Header extends React.Component {
       <Heading {...this.props}>
         <a className="anchor" id={slug} />
         {this.props.children}{' '}
-        <a className="hash-link" href={'#' + slug}>
+        <a className="hash-link" href={urljoin('#', slug)}>
```

Comment:
This will cause a bug. If slug is `verificar-la-instalacion`
We want `#verificar-la-instalacion`.
This will become `#/verificar-la-instalacion` (not what we want)

Reply:
Oh! I missed that I am updating this PR by fixing this issue